src/features/pages/partners/deriv-prime/meet-us-there/data.tsx

```diff
@@ -5,20 +5,6 @@ import { ExpoCardTypes } from './types'
 import { localize } from 'components/localization'
 
 export const expo_cards: ExpoCardTypes[] = [
-    {
-        title: '_t_iFX Expo Dubai_t_',
-        date: '_t_16-18 January 2024_t_',
-        image: (
-            <StaticImage
-                src="../../../../../images/common/deriv-prime/ifx_expo_dubai.png"
```

Comment:
Cool, please remove the static image from the repo as well

Reply:
yess, done